cc.craftos_pc.craftos_pc.yaml

```diff
@@ -99,6 +86,22 @@ modules:
           type: git
           tag-pattern: "^v([\\d.]+)$"
 
+  - name: craftos-desktop-assets
```

Comment:
no nee to create a new module for that either.

Reply:
I added another module to take advantage of caching to avoid building the craftos2 module every time I made a change on my local machine. Is there any advantage to unifying the modules or is it just style?

Reply:
you only needed to add the 128x128 icons and  you added files that are in the  upstream, you shouldn't.